pkgs/applications/misc/nut/default.nix

```diff
@@ -57,6 +57,10 @@ stdenv.mkDerivation rec {
 
   enableParallelBuilding = true;
 
+  # Add `cgi-bin` to the default list to avoid pulling in whole
+  # of `gcc` into build closure.
+  stripDebugList = [ "cgi-bin" "lib" "lib32" "lib64" "libexec" "bin" "sbin" ];
```

Comment:
Surprising that it overrides the default list

Maybe one day we should make that list additive only and then to exclude one of those directories stripEclude would be used https://github.com/NixOS/nixpkgs/pull/218783